.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+      - uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
```

Comment:
@dholth, is this step necessary since Miniconda is installed in the next step?

Reply:
Probably not.